CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # Changelog
+
+# 1.87.0
+* Feature - Add support for EBS Task Attach on Windows. [#4353](https://github.com/aws/amazon-ecs-agent/pull/4353)
+* Enhancement -  Update go module dependencies.
```

Comment:
still lingering around...

Reply:
oh sorry you mean remove the whole thing

Reply:
yes, sorry and thanks!